cmd/cosign/cli/options/attach.go

```diff
@@ -54,6 +55,8 @@ func (o *AttachSignatureOptions) AddFlags(cmd *cobra.Command) {
 			"when building the certificate chain for the signing certificate. "+
 			"Must start with the parent intermediate CA certificate of the "+
 			"signing certificate and end with the root certificate. Included in the OCI Signature")
+	cmd.Flags().StringVar(&o.TimeStampedSig, "timeStampedSignatureResponse", "",
```

Comment:
Can we use an acronym “—tsr” for the flag name instead?

Reply:
Agree i will change the flag name from --timeStampedSignatureResponse  to --tsr